lib/rules/selector-max-attribute/index.js

```diff
@@ -0,0 +1,75 @@
+"use strict"
+
+const isStandardSyntaxRule = require("../../utils/isStandardSyntaxRule")
+const isStandardSyntaxSelector = require("../../utils/isStandardSyntaxSelector")
+const parseSelector = require("../../utils/parseSelector")
+const report = require("../../utils/report")
+const ruleMessages = require("../../utils/ruleMessages")
+const validateOptions = require("../../utils/validateOptions")
+const resolvedNestedSelector = require("postcss-resolve-nested-selector")
+
+const ruleName = "selector-max-attribute"
+
+const messages = ruleMessages(ruleName, {
+  expected: (selector, max) => `Expected "${selector}" to have no more than ${max} attribute selector(s)`,
```

Comment:
I agree that this is nicer. However, I believe this style ("xxx(s)") is used for all `max` rules. Therefore I suggest we create a separate issue and make this change, at some point, across the board for the sake of consistency.